class.jetpack-jitm.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * Jetpack just in time messaging through out the admin
+ *
+ * @since 3.7.0
+ */
+class Jetpack_JITM {
+
+	/**
+	 * @var Jetpack_JITM
+	 **/
+	private static $instance = null;
+
+	static function init() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new Jetpack_JITM;
+		}
+
+		return self::$instance;
+	}
+
+	private function __construct() {
+		$jetpack_hide_jitm = Jetpack_Options::get_option( 'hide_jitm' );
+
+		if ( ! Jetpack::is_module_active( 'photon' ) && 'hide' != $jetpack_hide_jitm['photon'] ) {
+			add_action( 'admin_enqueue_scripts', array( $this, 'jitm_enqueue_files' ) );
```

Comment:
Since we're only using these on the media upload page, we probably shouldn't be adding them in on every admin page load